README.md

```diff
@@ -6,11 +6,155 @@
 
 ## Usage
 
-Coming soon!
+### TypeScript (NPM)
 
-## Contributing/Security
+> Repository: https://www.npmjs.com/package/cdk-monitoring-constructs
 
-See [CONTRIBUTING](CONTRIBUTING.md) for more information.
+Add the dependency to your `package.json`:
+
+```json
+{
+  "dependencies": {
+    "cdk-monitoring-constructs": "^0.0.11",
+
+    // peer dependencies
+    "constructs": "^3.3.69",  // 3.3.69 >= x < 4.0.0
+    "monocdk": "^1.123.0",    // 1.123.0 >= x < 2.0.0
+
+    // (your other dependencies)
+  }
+}
+```
+
+### Java (Maven)
+
+> Repository: https://mvnrepository.com/artifact/io.github.cdklabs/cdkmonitoringconstructs
+
+Add the following Maven project to your `pom.xml`:
+
+```xml
+<!-- https://mvnrepository.com/artifact/io.github.cdklabs/cdkmonitoringconstructs -->
+<dependency>
+    <groupId>io.github.cdklabs</groupId>
+    <artifactId>cdkmonitoringconstructs</artifactId>
+    <version>0.0.11</version>
+</dependency>
+```
+
+### Python (PyPi)
+
+> Repository: https://pypi.org/project/cdk-monitoring-constructs/
+
+TODO: describe usage
+
+### C# (Nugget)
+
+> Repository: https://www.nuget.org/packages/Cdklabs.CdkMonitoringConstructs/
+
+TODO: describe usage
+
+## Documentation
+
+* See [API](API.md) for complete auto-generated documentation.
```

Comment:
What do you think of just pointing to https://constructs.dev/packages/cdk-monitoring-constructs/ ?